testing/font-comic-neue/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=font-comic-neue
+_pkgname=comicneue
+pkgver=20160917
+pkgrel=0
+pkgdesc="Comic Neue is a font that fixes the shortcomings of Comic Sans"
+url="http://comicneue.com/"
+arch="noarch"
+license="OFL-1.1"
+depends="fontconfig"
+makedepends=""
+commit="216e1cca320e7e7886c9523f9afbb7972eb7a9f2"
+source="$_pkgname-$pkgver.zip::https://github.com/crozynski/comicneue/archive/$commit.zip"
+builddir="$srcdir/$_pkgname-$commit"
+subpackages="$pkgname-doc"
+options="!check"
+
+build() {
+	cd "$builddir"
+	return 0
+}
+
+package() {
+	cd "$builddir"
+	mkdir -p "$pkgdir/usr/share/fonts/$_pkgname"
+
+	for f in $(find OTF -type f -name "*.otf"); do
+		mv "$f" "$pkgdir"/usr/share/fonts/$_pkgname
+	done
```

Comment:
Why so complicated? Just `mv OTF/*.otf "$pkgdir"/usr/share/fonts/$_pkgname/` would be sufficient.